Godeps_windows

```diff
@@ -1,3 +1,4 @@
+github.com/Graylog2/go-gelf/gelf 76d60fc890684cff2fb65c631ed711d4af080592
```

Comment:
this is a fork of a fork of a fork of https://github.com/SocialCodeInc/go-gelf, why??